src/directive.js

```diff
@@ -19,7 +19,7 @@
       template: '<span ng-class="{\'is-placeholder\': placeholder && !editingValue}" ng-style="{\'max-width\': \'inherit\'}" >' +
         '<input ng-show="isEditing" ng-blur="onInputBlur()" ng-keydown="onKeyPress($event)" ' +
           'ng-model="editingValue" placeholder="{{placeholder}}" type="text" ' +
-          'pu-elastic-input pu-elastic-input-minwidth="auto" pu-elastic-input-maxwidth="100%" />' +
+          'pu-elastic-input pu-elastic-input-minwidth="100%" pu-elastic-input-maxwidth="100%" />' +
```

Comment:
I wonder if we're getting any benefit from elastic-input, since we're setting the min/max both to 100%. But I don't remember the benefits that it provided (it was added by an intern).